### ISSUE TITLE
Updated rule: `audio` or `video` has no audio that plays automatically

### ISSUE THREAD

**Issue by Jym77** [w3c/wcag-act#490]
The ACT-Rules community group has completed work on "`audio` or `video` has no audio that plays automatically". We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: `audio` or `video` has no audio that plays automatically
**Rule id**: 80f0bf
**Link to the rule**: https://act-rules.github.io/rules/80f0bf
**License**: W3C Software And Document Notice And License
**Implementors**:
- axe-core
- Sortsite

If you'd like to contact me, please e-mail me at jym@siteimprove.com. (optional: You can also reach me on the W3C Community Slack, my name there is Jean-Yves Moyen).
This update follows feedback from the Task Force on a preliminary version: #456.

**Comment by maryjom** [comment]
[Survey](https://www.w3.org/2002/09/wbs/93339/ACTAudioPlaysAuto/results) in progress until 22 October.

**Comment by maryjom** [comment]
Survey is complete. Needs new issue opened with comments. See discussions of the survey in our meetings: 

- [minutes from 15 October](https://www.w3.org/2020/10/15-wcag-act-minutes.html#item03)
- [minutes from 22 October](https://www.w3.org/2020/10/22-wcag-act-minutes.html#item04)
